### ISSUE TITLE
[FEATURE] Download all attached files at once

### ISSUE THREAD

**Issue by jobenjada** [formbricks/formbricks#3323]
### Is your feature request related to a problem? Please describe.

People have to download all files individually.

### Describe the solution you'd like

<img width="716" alt="image" src="https://github.com/user-attachments/assets/cc3b12fc-1183-49fe-9103-9b41cb5302e6">

THIS FEATURE IS DEPENDENT ON GETTING THIS  PR MERGED FIRST! PLEASE DONT START WORKING ON IT YET.

https://github.com/formbricks/formbricks/pull/3272

You can self-assign tho :)

**Comment by ppmpreetham** [comment]
Hi. Can I please work on this issue?  :)

**Comment by jobenjada** [comment]
Raju is on it, sorry :)

**Comment by Khaan25** [comment]
@jobenjada I would like to work on this issue?

**Comment by silentboy-07** [comment]
The current issue is that users need to download files one by one. I propose a solution that enables downloading all files at once. However, this feature is dependent on the merge of PR #3272, so please assign this to me once that PR is merged.

**Comment by Khaan25** [comment]
@jobenjada, Can I work on this? :)

**Comment by risrajawat** [comment]
It seems this issue is still not resolved. Could you please assign it to me?

/assign

**Comment by jobenjada** [comment]
hey @risrajawat thanks for offering support but as of now we don't have the capacity to facilitate community contributions.

**Comment by Ayushkr3864** [comment]
@jobenjada is this still open? as I want to work on it

**Comment by jobenjada** [comment]
@Ayushkr3864 it's something we want to build on top of the new background job infra structure, so we need to wait for this to be reviewed and merged into main
